src/components/Header.jsx

```diff
@@ -46,6 +46,8 @@ class Header extends Component {
           </Link>
           <div className="block lg:hidden">
             <button
+              aria-labelledby="header--nav-menu"
+              aria-pressed={!menuClosed}
```

Comment:
Argh, totally my bad, I’ve led you astray with #36. `aria-pressed` should be `aria-expanded` in this context.